src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC } from "react";
+import { Loader } from "@mantine/core";
+import { useGetTodosQuery } from "../../redux/apiSlice";
+import { Todo } from "../Todo/Todo";
+import { StyledTodoWrapper } from "./TodoListStyles";
+
+export const TodoList: FC = () => {
```

Comment:
Když jsem si to zapnul poprvé tak jsem neměl puštěný BE a nikde se nezobrazila žádná chyba. Ještě bych tam přidal nějaký lepší handling, abych jasně věděl že mi selhal BE.